### PR TITLE
Feature/alert modal

### LINKED ISSUE #68: JSX/CSS: Alert modal component
- [ ] Build out Alert modal component
- [ ] Format and style modal display
- [ ] Connect modal with SMS messaging functionality

### PR DESCRIPTION
**Pull request by pcmueller**
## Type of Change
- [x] JSX
- [ ] HTML
- [ ] JavaScript
- [x] CSS
- [x] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

## Please describe the change made
Alert Message modal

## Please give a short description of the changes that were made below

Feature:
- Built out `Alert` modal component and completed styling and basic open/close functionality, including timeout
- Lifted "emergency" state up to CurrentTrip component, and added useEffect() function that will run when Alert modal closes - we can add SMS alert messaging functionality here (CurrentTrip lines 70-75).

## How has this been tested?
- [x] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman

https://user-images.githubusercontent.com/51062974/126086459-633e6538-a252-4d82-a249-d1c657f925a8.mov

### CODE REVIEW

File: src/Components/Alert/Alert.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            onComplete={handleModalClose}
+          >
+          </CountdownCircleTimer>
+          <NavLink className='alert-exit-link' exact to='/'>
+            <i type='button' onClick={handleModalClose} className="fas fa-times fa-2x alert-exit-btn"></i>
+          </NavLink>
+        </div>
+        <h2 className='alert-title'>!ALERT!</h2>
+        <p className='alert-message'>
+          Your contact has been notified that your trip was not completed in time.
```
Comment (bhohnco) [inline src/Components/Alert/Alert.js:46]: This looks so good, well done Pete!

File: src/Components/CurrentTrip/CurrentTrip.js [hunk 2]
```diff
@@ -54,24 +57,37 @@ function CurrentTrip({ user, eta }) {
...
-      setModalIsOpen(true);
+      setEtaModalIsOpen(true);
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [tripIsActive]);
 
+  useEffect(() => {
+    if (emergency) {
+      setAlertModalIsOpen(true);
+      // invoke function to send Alert message here
```
Comment (bhohnco) [inline src/Components/CurrentTrip/CurrentTrip.js:73]: Thanks for the heads up Pete, well get this tied in here.

File: src/Components/CurrentTrip/CurrentTrip.js [hunk 3]
```diff
@@ -153,13 +169,22 @@ function CurrentTrip({ user, eta }) {
...
+      {!tripIsActive &&
+        <AddTime 
+          setExtension={setExtension}
+          setEtaSeconds={setEtaSeconds}
+          modalIsOpen={etaModalIsOpen} 
+          closeModal={closeEtaModal}
+          setEmergency={setEmergency}
+        />
+      }
+      {emergency &&
```
Comment (carolineshowscode) [inline src/Components/CurrentTrip/CurrentTrip.js:181]: Great job adding this alert/sad path into our current trip

**Review by carolineshowscode — APPROVED** [review]
@pcmueller two CY tests failing to be fixed before branch merge

**Review by gaelyn — APPROVED** [review]
Not sure what all is going on here, but I did take a look at it and we chatted about the alerts as a group. Y'all are crushing it!

**Pull request merged**